core-bundle/contao/widgets/SerpPreview.php

```diff
@@ -41,7 +41,7 @@ public function generate()
 		}
 
 		$id = $model->id;
-		$title = StringUtil::substr(str_replace(array('&nbsp;', '&shy;'), array(' ', ''), $this->getTitle($model)), 64);
+		$title = StringUtil::substr(str_replace(array('[nbsp]', '[-]'), array(' ', ''), $this->getTitle($model)), 64);
```

Comment:
How can this work? `[nbsp]` and `[-]` should never come from the database because it is converted to entities before storing it.

Reply:
I think we have to revert this line and enable basicEntities in https://github.com/contao/contao/blob/afa626a5e85a5d95e86e9764b0a94707d4d051a2/core-bundle/contao/dca/tl_page.php#L220-L226

Reply:
Changed in 7b0dd4ddfcb59e5a9b4e21baab8831a9c1d96841. Thanks for pointing me in the right direction.